getting_started/first_3d_game/08.score_and_replay.rst

```diff
@@ -292,17 +292,16 @@ scene.
 Adding music
 ------------
 
-To add music that plays continuously in the background, we're going to use
-another feature in Godot: :ref:`autoloads <doc_singletons_autoload>`.
+To continuously play background music, we introduce the :ref:`Autoloads <autoloads_versus_internal_nodes>` function.
```

Comment:
```suggestion
To continuously play background music, we introduce the :ref:`Autoloads <doc_singletons_autoload>` function.
```

The link should go back to directing people towards the singletons page. having two "autoloads" links going to 2 different places would be confusing for users, also I think that causes a build issue (current build issue is because there's no "doc_" in this.